js/navigation.js

```
@@ -289,62 +305,64 @@ class DropdownManager {
-            case 'Escape':
-                event.preventDefault();
-                this.closeAllDropdowns();
-                const trigger = this.activeDropdown?.querySelector('.dropdown-trigger');
-                if (trigger) {
-                    trigger.focus();
-                }
-                break;
+        case 'Escape':
+            event.preventDefault();
+            this.closeAllDropdowns();
+            const trigger = this.activeDropdown?.querySelector('.dropdown-trigger');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)